src/domain/repository/cellTowerRepository.js

```diff
@@ -44,4 +44,4 @@ export class CellTowerRepository {
     }
 }
 
-const shouldLookup = (result) => null == result || result.status == 'Pending'
\ No newline at end of file
+const shouldLookup = (result) => !result || result.status == 'Pending'
```

Comment:
A better construct is advised for checking existence.

For instance, `[] == false` evaluates to `true`.

With this, strict equality operators (`===` and `!==`) perform more consistently than their non-strict counterparts.

Reply:
I agree